node/pkg/watchers/aptos/watcher.go

```diff
@@ -41,11 +41,13 @@ type (
 )
 
 var (
+	//nolint:exhaustruct
```

Comment:
Can you add a reason for these two?

Reply:
Yes, I added comments in several of the nolints that I added in.